keyboards/handwired/mingwahlu/mingnumpad/rules.mk

```diff
@@ -0,0 +1,23 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+BOOTLOADER = caterina
```

Comment:
One or the other, not both. 

```suggestion
BOOTLOADER = caterina
```

```suggestion
BOOTLOADER = atmel-dfu
```

Reply:
I have changed all 3 folders to contain only 1 of the follow(BOOTLOADER = atmel-dfu)